### PR TITLE
Fix doc

### PR DESCRIPTION
**Pull request by k-takata**

### CODE REVIEW

File: runtime/doc/debugger.txt [hunk 1]
```diff
@@ -114,7 +114,7 @@ other settings apply.
 
 The debugger features were added for use with Sun's Visual WorkShop Integrated
 Programming Environment (ipe).  However, they were done in as generic a manner
```
Comment (k-takata) [inline runtime/doc/debugger.txt:116]: Is "generic a manner" a mistake of "a generic manner"?
(Not fixed in this patch.)

File: runtime/doc/debugger.txt [hunk 2]
```diff
@@ -114,7 +114,7 @@ other settings apply.
...
-as possible so that integration with other debuggers could also these
+as possible so that integration with other debuggers could also use these
```
Comment (k-takata) [inline runtime/doc/debugger.txt:117]: "use" was missing.

File: runtime/doc/options.txt [hunk 3]
```diff
@@ -517,10 +517,11 @@ Examples:
...
 The second form (this is compatible with some versions of Vi):
 
-	[text]{white}{vi:|vim:|Vim:|ex:}[white]se[t] {options}:[text]
+	[text{white}]{vi:|vim:|Vim:|ex:}[white]se[t] {options}:[text]
 
-[text]			any text or empty
-{white}			at least one blank character (<Space> or <Tab>)
+[text{white}]		empty or any text followed by at least one blank
+			character (<Space> or <Tab>); "ex:" always requires at
+			least one blank character
```
Comment (k-takata) [inline runtime/doc/options.txt:524]: I think that the discussion in
https://groups.google.com/d/topic/vim_dev/-hHiBLao06Y/discussion
should be also applied to the second form.

File: runtime/doc/options.txt [hunk 4]
```diff
@@ -6752,8 +6753,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	option is on, dots in file names are replaced with underscores when
 	adding an extension (".~" or ".swp").  This option is useful
 	when editing files on an MS-DOS compatible filesystem, e.g., messydos
-	or crossdos.  When running the Win32 GUI version under Win32s, this
-	option is always on by default.
+	or crossdos.
```
Comment (k-takata) [inline runtime/doc/options.txt:6756]: Win32s is no longer supported.

File: runtime/doc/options.txt [hunk 5]
```diff
@@ -7288,8 +7288,8 @@ A jump table for the options with a short description can be found at |Q_op|.
 <	Beware that an expression is evaluated each and every time the status
 	line is displayed.
 				*stl-%{* *g:actual_curbuf* *g:actual_curwin*
-	While evaluationg %{} the current buffer and current window will be
-	set temporarily to that of the window (and buffer) whose statusline is
+	While evaluating %{} the current buffer and current window will be set
```
Comment (k-takata) [inline runtime/doc/options.txt:7291]: Fixed typo.

File: runtime/doc/quickfix.txt [hunk 6]
```diff
@@ -236,8 +236,7 @@ processing a quickfix or location list command, it will be aborted.
 			e.g., a compiler will not compile the same file again,
 			`git commit` will abort the committing process, `fc`
 			(built-in for shells like bash and zsh) will not
-			execute the command, etc.  will not compile the same
-			file again.
+			execute the command, etc.
```
Comment (k-takata) [inline runtime/doc/quickfix.txt:239]: Remove wrongly duplicated part.

File: runtime/doc/quickfix.txt [hunk 7]
```diff
@@ -1108,7 +1107,7 @@ id-utils) in a similar way to its compiler integration (see |:make| above).
 			allowed with |:bufdo|.
 			An example that uses the argument list and avoids
 			errors for files without matches: >
-                                :silent argdo try
+				:silent argdo try
```
Comment (k-takata) [inline runtime/doc/quickfix.txt:1110]: Use tab instead of space.

File: runtime/doc/debugger.txt (hunk 1 above)
Comment (gdupras) [reply runtime/doc/debugger.txt:116]: No, I believe it is correct in the context of the rest of the sentence. If you search online for "generic a manner", you will find several instances of "as generic a manner as possible".

**Comment by brammool** [comment]
I'll include it, thanks.

**Pull request closed without merging**